### PR TITLE
WIP: Cfs/rh cf rp dev env working sessions

### PR DESCRIPTION
**Pull request by coreyperkins**
Work-in-progress: Do not merge. Using to track working sessions between RH and my org.

### Which issue this PR addresses:

Fixes

### CODE REVIEW

**Review by bryanro92 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -123,16 +123,22 @@ pyenv:
 		sed -i -e "s|^dev_sources = $(PWD)$$|dev_sources = $(PWD)/python|" ~/.azure/config
```
Comment (bryanro92) [inline Makefile:123]: sorry for the noise - needed a pr w/o any comments/reviews and am testing something

Comment (bryanro92) [reply Makefile:123]: 2

File: Makefile [hunk 2]
```diff
@@ -123,16 +123,22 @@ pyenv:
...
```
Comment (bryanro92) [inline Makefile:124]: 3

**Review by bryanro92 — CHANGES_REQUESTED** [review]
sorry for noise... testing a feature

**Comment by coreyperkins** [comment]
Closing this PR, it was only used for experimentation purposes.

**Pull request closed without merging**